runtime/src/main/resources/application.yml

```diff
@@ -17,12 +17,6 @@
 deployment:
   file: io/syndesis/dao/deployment.json
 
-resteasy:
```

Comment:
Actually this changes from explicit JAX-RS class specified in config which requires no scanning at all, to default behaviour which is to scan spring registry for all JAX-RS application beans. This allows for easier configuration at the expense of a few ms more scanning time (no full classpath scan so fast).